dist/merge/index.js

```diff
@@ -9104,6 +9121,10 @@ class DefaultGlobber {
                 if (!stats) {
                     continue;
                 }
+                // Hidden file or directory?
+                if (options.excludeHiddenFiles && path.basename(item.path).match(/^\./)) {
```

Comment:
what about gh-pages' `.nojekyll`?

Reply:
Thank you for the feedback, for now we are going to go ahead with the 'all in' option as starting to pick and choose common things is a risk we start to go back to 'too many' common things and expose folks to the risk of mistakes causing issues. 
I get the idea though, maybe in the future we could look at 'more flags' for "all-common-none", but that will be something we need to revisit at a later date so we could get a better view on what common would be :)

Reply:
Well at least there should probably be a black/whitelist option rather than a `bool` methinks

Reply:
`path` is effectively that allowlist/denylist option - you can allow hidden files and then specify in `path` which to exclude

https://github.com/actions/upload-artifact#upload-using-multiple-paths-and-exclusions